### DISCUSSION TITLE
Notes (todo notes) support

### DISCUSSION THREAD

**Discussion by Solarunit** [immich-app/immich#3089] [General]
I know immich is for photo and video but what about (markdown) notes and maybe todo notes support?
I think it would be nice to have.

**Comment by colin715** [comment]
This seems way outside the scope of Immich

**Reply by Solarunit** [reply]
What if you could attach such markdown notes to albums or some particular photos (half joking)
But I agree notes are not directly related to photos..

**Reply by lukehmcc** [reply]
Just use Joplin?
